index.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/398c36776a.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="header_hero_bg">
+            <nav class="header_nav container">
+
+                <a href="#default" class="logo"></a>
```

Comment:
Lepiej wrzucić logo jako `img` - wtedy do `alt=""` można wrzucić parę słów kluczowych. Dla SEO będzie pomocne ;)

PS. To nie jest favicon, logo jest w pliku `logo.svg`